SYCL/BFloat16/bfloat_hw.cpp

```diff
@@ -0,0 +1,50 @@
+// RUN: %clangxx -fsycl %s -o %t.out
+// RUN: %HOST_RUN_PLACEHOLDER %t.out
```

Comment:
```suggestion
```

Host device is going to be removed soon. Moreover, you don't even use any devices in the test, so no need for any special placeholders

Reply:
Maybe add a `single_task` to verify it works on device as well then?

Reply:
Should I just say
`// RUN: %t.out`
?

Reply:
> Maybe add a single_task to verify it works on device as well then?

Yes, good idea. But this is E2E regression test for particular problem I found on host. Test coverage can be improved in separate patches, I believe.